app/assets/javascripts/components/profile_card.es6.jsx

```diff
@@ -1,12 +1,26 @@
 class ProfileCard extends React.Component {
 
+  static getpropTypes() {
+    return {
+      cardName: React.PropTypes.string,
+      cardBody: React.PropTypes.string,
+    }
+  }
+
+  static get defaultProps() {
+    return {
+      cardName: '',
+      cardBody: '',
+    }
```

Comment:
Semicolon after the `}` here! I'll be adding a js-linter soon (sorry haven't done it yet).